pkg/util/deployment/deployment.go

```diff
@@ -28,6 +28,11 @@ import (
 	podutil "k8s.io/kubernetes/pkg/util/pod"
 )
 
+const (
+	// The revision annotation of a deployment's replication controllers which records its rollout sequence
+	RevisionAnnotation = "rollout.kubectl.kubernetes.io/revision"
```

Comment:
These revisions aren't generated by kubectl. They are generated by deployment, so deployment.kubernetes.io/revision would be more appropriate.